### PR TITLE
Release figure 4.0.0 post

### PR DESCRIPTION
**Pull request by will-moore**
Announcement of Figure release 4.0.0 - aiming for next Monday.
Added explanation of Big-Images and Stroke-width changes. Make sense??

cc @jburel

### CODE REVIEW

File: _posts/2018-05-21-figure-4-0-0.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+It includes changes to the way ROI stroke-widths are displayed.
+Previously, ROI stroke-widths were measured in image pixels so that
+stroke-widths decreased as an image panel was zoomed out.
+However, this meant that lines of similar width can appear at different
+thicknesses on different images. It also meant that on "big" images that
+were zoomed out a long way, lines would disappear completely.
+
+Now, stroke-widths are measured by page points so all lines of a
+similar stroke-width will appear the same thickness on the page,
+independent on the size or zoom of the panel they are on.
```
Comment (mtbc) [inline _posts/2018-05-21-figure-4-0-0.md]: s/on/of/

File: _posts/2018-05-21-figure-4-0-0.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+When older figures are opened with OMERO.figure 4.0.0, stroke-widths
+will be updated so that the appearance of ROI lines remains the same.
+
+Other features:
+
+* Allow loading Polygons from OMERO to add to images
```
Comment (mtbc) [inline _posts/2018-05-21-figure-4-0-0.md]: We tend to follow https://en.oxforddictionaries.com/punctuation/bullet-points where sentence bullet-points start with a capital and end with a full stop and non-sentences have neither.

File: _posts/2018-05-21-figure-4-0-0.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+* Open-with Figure and adding Images now preserves the order of images
```
Comment (mtbc) [inline _posts/2018-05-21-figure-4-0-0.md]: There is an initial capital for one "Images" but not for any of the other images in the list: are those special?

**Comment by mtbc** [comment]
Still have a mix of image/Image capitalization and re. https://en.oxforddictionaries.com/punctuation/bullet-points now a mix of bullet points with / without end period.

**Comment by jburel** [comment]
@mtbc better? 
I have also updated https://github.com/ome/omero-figure/pull/287

**Comment by will-moore** [comment]
Looks good.

**Comment by jrswedlow** [comment]
Suggested changes:

This is a major release that adds support for "big" tiled Images.  This means that OMERO.figure can now handle WSI image data or other images with large XY sizes.

This release also includes changes to the way ROI stroke-widths are displayed.

**Pull request merged**